recipes-overlayed/kselftests/kselftests_4.12-rc7.bb

```diff
@@ -47,6 +49,7 @@ PACKAGE_BEFORE_PN = " \
 	${PN}-bpf \
 	${PN}-breakpoints \
 	${PN}-capabilities \
+	${PN}-cpufreq \
```

Comment:
you've added cpufreq and splice, but they aren't the only tests addition. intel-pstate is another one and is missing. kselftests recipe isn't for ARM only.